template/razor_repl_expr.go

```diff
@@ -80,7 +83,7 @@ func expressionParserInternal(repl replacement, match string, skipError, interna
 			sep, slicer, limit2 = ":", "slice", true
 		}
 		values := strings.Split(indexExpr, sep)
-		if limit2 && len(values) > 2 {
+		if !debugMode && limit2 && len(values) > 2 {
```

Comment:
Why not print this when in debugMode?

Reply:
Again, I just restored the feature as it was before. The debug mode is only enabled under certain circumstance.

Reply:
It just seems like debugMode should print more stuff, not less

Reply:
In debug mode, we process code that as already been processed. The idea is to not issue the same error multiple times.